Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -562,6 +570,34 @@ theorem not_summable_of_ratio_test_tendsto_gt_one {α : Type*} [SeminormedAddCom
   rwa [← le_div_iff (lt_of_le_of_ne (norm_nonneg _) h₁.symm)]
 #align not_summable_of_ratio_test_tendsto_gt_one not_summable_of_ratio_test_tendsto_gt_one
 
+section NormedDivisionRing
+
+variable [NormedDivisionRing α] [CompleteSpace α] {f : ℕ → α}
+
+/-- If a power series converges at `w`, it converges absolutely at all `z` of lesser norm. -/
+theorem summable_power_of_norm_lt {w z : α}
```

Comment:
I think `powerSeries` in the name would be better (even though the statement does not mention terms of type `PowerSeries`).

Reply:
That is a matter for the dependency #9955 (but I have applied the rename there).